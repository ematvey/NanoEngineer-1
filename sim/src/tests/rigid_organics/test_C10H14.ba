LENGTH 0 1 1.54462746318
LENGTH 0 2 1.54500582523
LENGTH 0 10 1.09633252255
LENGTH 0 11 1.09634529232
LENGTH 1 3 1.54352453819
LENGTH 1 4 1.53700130124
LENGTH 1 12 1.10206351904
LENGTH 2 5 1.53712393775
LENGTH 2 6 1.54322584219
LENGTH 2 13 1.10132783493
LENGTH 3 7 1.54604333704
LENGTH 3 14 1.1026957876
LENGTH 3 15 1.10002090889
LENGTH 4 9 1.54440830094
LENGTH 4 16 1.10245226654
LENGTH 4 17 1.09803005423
LENGTH 5 9 1.54454815399
LENGTH 5 18 1.09702370075
LENGTH 5 19 1.10021861464
LENGTH 6 7 1.54454167959
LENGTH 6 20 1.09827136902
LENGTH 6 21 1.1035411184
LENGTH 7 8 1.53023200855
LENGTH 7 22 1.09913329492
LENGTH 8 9 1.54568366751
LENGTH 8 23 1.10181940444
LENGTH 8 24 1.10165012595
LENGTH 9 25 1.09931160278
ANGLE 0 1 3 109.294178581
ANGLE 0 1 4 109.302982202
ANGLE 0 1 12 109.46858388
ANGLE 0 2 5 109.505672408
ANGLE 0 2 6 109.255878119
ANGLE 0 2 13 109.638919664
ANGLE 1 0 2 109.670221845
ANGLE 1 0 10 110.178669168
ANGLE 1 0 11 109.937999728
ANGLE 1 3 7 109.552428631
ANGLE 1 3 14 110.192946177
ANGLE 1 3 15 109.907120591
ANGLE 1 4 9 109.779434632
ANGLE 1 4 16 110.140143692
ANGLE 1 4 17 109.92987692
ANGLE 2 0 10 110.429656696
ANGLE 2 0 11 110.025732593
ANGLE 2 5 9 109.797263792
ANGLE 2 5 18 109.919373776
ANGLE 2 5 19 110.139336376
ANGLE 2 6 7 109.736479811
ANGLE 2 6 20 110.240590801
ANGLE 2 6 21 110.218377813
ANGLE 3 1 4 109.480602987
ANGLE 3 1 12 109.624602575
ANGLE 3 7 6 108.980501017
ANGLE 3 7 8 109.537850299
ANGLE 3 7 22 109.378947556
ANGLE 4 1 12 109.655290843
ANGLE 4 9 5 109.366199171
ANGLE 4 9 8 109.151654453
ANGLE 4 9 25 109.567190633
ANGLE 5 2 6 109.129111482
ANGLE 5 2 13 109.658414316
ANGLE 5 9 8 109.450545059
ANGLE 5 9 25 109.511888077
ANGLE 6 2 13 109.637411156
ANGLE 6 7 8 109.912852775
ANGLE 6 7 22 109.506494424
ANGLE 7 3 14 110.174049591
ANGLE 7 3 15 110.131387659
ANGLE 7 6 20 110.2669458
ANGLE 7 6 21 109.850239957
ANGLE 7 8 9 109.569933628
ANGLE 7 8 23 110.323330327
ANGLE 7 8 24 110.11390783
ANGLE 8 7 22 109.508585976
ANGLE 8 9 25 109.77859147
ANGLE 9 4 16 110.01948388
ANGLE 9 4 17 110.119989827
ANGLE 9 5 18 110.180208929
ANGLE 9 5 19 109.979702878
ANGLE 9 8 23 110.095896595
ANGLE 9 8 24 109.789814774
ANGLE 10 0 11 106.555650156
ANGLE 14 3 15 106.849978126
ANGLE 16 4 17 106.81021367
ANGLE 18 5 19 106.782173108
ANGLE 20 6 21 106.481902101
ANGLE 23 8 24 106.914258532
