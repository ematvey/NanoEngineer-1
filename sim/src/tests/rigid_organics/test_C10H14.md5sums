stdout EA0CCFF8759B26A6C0617064A1776BD5
stderr D41D8CD98F00B204E9800998ECF8427E
