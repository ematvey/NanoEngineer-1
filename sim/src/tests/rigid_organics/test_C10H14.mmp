mmpformat 050502 required; 050706 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (7.304694) (-0.000000, -0.000000, -0.000000) (1.000000)
egroup (View Data)
group (C10H14)
info opengroup open = True
mol (C10H14.pdb) def
atom 1 (6) (-1939, 817, 577) def
atom 2 (6) (-1462, -1157, 585) def
bond1 1
atom 3 (6) (70, 1327, -321) def
bond1 1
atom 4 (6) (-74, -1550, 1286) def
bond1 2
atom 5 (6) (-1038, -1598, -1135) def
bond1 2
atom 6 (6) (45, 908, -1717) def
bond1 3
atom 7 (6) (1419, 1055, 989) def
bond1 3
atom 8 (6) (1269, -363, 1219) def
bond1 4 7
atom 9 (6) (1523, -684, -774) def
bond1 8
atom 10 (6) (535, -640, -1752) def
bond1 5 6 9
atom 11 (1) (-2437, 864, 276) def
bond1 1
atom 12 (1) (-1568, 1504, 1037) def
bond1 1
atom 13 (1) (-2169, -1883, 1068) def
bond1 2
atom 14 (1) (-415, 2398, 519) def
bond1 3
atom 15 (1) (-144, -2461, 1501) def
bond1 4
atom 16 (1) (-290, -864, 2851) def
bond1 4
atom 17 (1) (-523, -2592, -1035) def
bond1 5
atom 18 (1) (-1750, -1455, -1191) def
bond1 5
atom 19 (1) (822, 1360, -2011) def
bond1 6
atom 20 (1) (-966, 1488, -2195) def
bond1 6
atom 21 (1) (1627, 1750, 1039) def
bond1 7
atom 22 (1) (660, 1030, 1641) def
bond1 7
atom 23 (1) (1843, -479, 1325) def
bond1 8
atom 24 (1) (2804, -731, -293) def
bond1 9
atom 25 (1) (1328, -2143, -740) def
bond1 9
atom 26 (1) (608, -1142, -2444) def
bond1 10
egroup (C10H14)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part C10H14

