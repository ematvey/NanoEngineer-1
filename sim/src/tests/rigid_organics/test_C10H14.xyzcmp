26
C10H14.mmp
C -1.586000 0.651000 0.418000
C -1.397000 -0.882000 0.410000
C -0.265000 1.340000 0.009000
C -0.285000 -1.264000 1.410000
C -0.995000 -1.337000 -1.002000
C 0.140000 0.878000 -1.400000
C 0.844000 0.948000 1.008000
C 1.040000 -0.584000 0.995000
C 1.435000 -1.047000 -0.409000
C 0.328000 -0.655000 -1.414000
H -2.394000 0.936000 -0.266000
H -1.890000 0.985000 1.417000
H -2.341000 -1.370000 0.702000
H -0.400000 2.433000 0.016000
H -0.158000 -2.359000 1.438000
H -0.570000 -0.950000 2.425000
H -0.879000 -2.433000 -1.029000
H -1.789000 -1.084000 -1.717000
H 1.069000 1.376000 -1.704000
H -0.630000 1.169000 -2.130000
H 1.784000 1.454000 0.750000
H 0.577000 1.283000 2.025000
H 1.830000 -0.859000 1.708000
H 2.394000 -0.593000 -0.706000
H 1.580000 -2.139000 -0.421000
H 0.611000 -0.981000 -2.425000
