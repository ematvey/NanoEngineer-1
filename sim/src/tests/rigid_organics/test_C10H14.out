======= test_C10H14.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
cutover to fine tolerance at 1
